### DISCUSSION TITLE
Install from dev branch: Comlandr

### DISCUSSION THREAD

**Discussion by sgaichas** [NOAA-EDAB/comlandr#65] [Announcements]
see sarahs comment

**Comment by sgaichas** [comment]
For anyone working on `comlandr` note that we are working from the `dev` branch. Please do not install from the `master` branch as this is outdated

**Comment by andybeet** [comment]
@sgaichas you will need to update your local clone of comlandr. I have updated master from dev and renamed the `master` to `main`
You'll probably be told to do this once you visit th repo but just in case 

```
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```
